src/detail-page/components/TabJobContent/JobTab.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import { Button, CircularProgress, Input } from '@material-ui/core';
+import JobListItem from './JobListItem';
+import useFetchJobs from './useFetchJobs';
+import Job from '../../../data/Jobs';
+import hiringPicture from '../../../icons/hiring.svg';
+
+const SearchInput = styled(Input)`
+  width: 250px;
+  height: 35px;
+  border: 1px #CFCFCF solid;
+  margin: ${props => props.theme.spacing(5)}px 0 ${props => props.theme.spacing(1.5)}px ${props => props.theme.spacing(1.5)}px ;
+  padding-left: 0 ${props => props.theme.spacing(1.5)}px 
+`;
+
+const SearchButton = styled(Button)`
+  width: 100px;
+  margin: ${props => props.theme.spacing(1.5)}px;
+  display: block;
+  color: white;
+`;
+
+const HiringImage = styled('object')`
+  width: 250px;
+  float: right;
+  margin: 0 ${props => props.theme.spacing(10)}px ${props => props.theme.spacing(10)}px 0;
```

Comment:
Positioning an element using big margin value is not the best solution because there is a big chance that you will have some problems with that in the future, or at least you will have to overwrite it for smaller screens. Here the value is 80px so for example for mobile view it's way to big. I would probably try to use flex-box here and align the element vertically and horizontally.

Reply:
done